bin/datatool/src/args.rs

```diff
@@ -0,0 +1,187 @@
+//! Command line arguments for Strata's `datatool` binary.
+
+use std::path::PathBuf;
+
+use argh::FromArgs;
+use bitcoin::Network;
+use rand::rngs::OsRng;
+
+/// Args.
+#[derive(FromArgs)]
+pub(crate) struct Args {
+    #[argh(option, description = "network name [signet, regtest]", short = 'b')]
+    pub(crate) bitcoin_network: Option<String>,
+
+    #[argh(
+        option,
+        description = "data directory (unused) (default cwd)",
+        short = 'd'
+    )]
+    pub(crate) datadir: Option<PathBuf>,
+
+    #[argh(subcommand)]
+    pub(crate) subc: SubcommandGen,
+}
+
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand)]
+pub(crate) enum SubcommandGen {
+    Seed(SubcSeed),
+    SeqPubkey(SubcSeqPubkey),
+    SeqPrivkey(SubcSeqPrivkey),
+    OpXpub(SubcOpXpub),
+    Params(SubcParams),
+}
```

Comment:
The `Gen` suffix on this is kinda redundant, it's not *just* for generating stuff.

Reply:
Good point done in 0d73020d7841b9f30112b2a58684f1ab89f3ddf6